app/services/dos_to_unix_lines.rb

```diff
@@ -0,0 +1,16 @@
+class DosToUnixLines
+	def initialize path_to_file, path_to_result
+		@path1 = path_to_file
```

Comment:
The naming of these variables is not great. Something like `source_filepath` and `destination_filepath` will make more sense.